src/main/routes/home.ts

```diff
@@ -1,7 +1,92 @@
+import * as fs from 'fs';
+import * as path from 'path';
+
+import { SecureRequester } from './../assets/js/secureRequester';
+
+import axios from 'axios';
 import { Application } from 'express';
 
+const multer = require('multer');
+
 export default function (app: Application): void {
-  app.get('/', (req, res) => {
-    res.render('home');
+  const upload = multer({ dest: 'uploads/' });
+
+  app.get('/', async (req, res) => {
+    try {
+      res.render('home', { 'output': 'Ready...' });
+    } catch (error) {
+      console.error('Error making request:', error);
+      res.render('home', {});
+    }
+  });
+
+  app.post('/', upload.single('myfile'), async (req, res) => {
```

Comment:
This is massive. Please separate these marks into methods within the endpoint

Reply:
separated out now